apps/meteor/app/lib/server/functions/getRoomsWithSingleOwner.ts

```diff
@@ -64,11 +64,11 @@ export async function getSubscribedRoomsForUserWithDetails(
 				}
 
 				// If there's no subscriber available to be the new owner and it's not a public room, we can remove it.
-				if (!roomData.shouldChangeOwner && roomData.t !== 'c') {
+				if (!roomData.shouldChangeOwner) {
 					roomData.shouldBeRemoved = true;
 				}
 			}
-		} else if (roomData.t !== 'c') {
```

Comment:
This is probably by design. Should we really change this behavior?

Reply:
Reopening this comment to confirm that it is by design and that the behavior should not be changed in the scope of this PR.

Reply:
Fixed, @totoi690 will add it to another PR